lesson07/assignment/coffee-project/src/main.js

```diff
@@ -0,0 +1,20 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import { BootstrapVue } from 'bootstrap-vue'
+
+// Import Bootstrap and BootstrapVue CSS files (order is important)
```

Comment:
You need to add a line to import boostrap.  Without this you only get the bootstrap styles and non of the javascript functionality.  I wouldn't bother with BootstrapVue unless you have a good reason to use their prebuilt components.

`import 'bootstrap'`